src/main/java/com/ays/user/model/entity/UserEntity.java

```diff
@@ -91,11 +96,16 @@ public Map<String, Object> getClaims() {
         claims.put(AysTokenClaims.USER_FIRST_NAME.getValue(), this.firstName);
         claims.put(AysTokenClaims.USER_LAST_NAME.getValue(), this.lastName);
         claims.put(AysTokenClaims.INSTITUTION_ID.getValue(), this.institutionId);
+        claims.put(AysTokenClaims.USER_ID.getValue(), this.id);
```

Comment:
Bu kısmı en üste taşımamız daha okunaklı olmasını sağlayabilri

Reply:
Bunu daha anlaşılması için yukarı aldım.